### PR TITLE
README section tweaks

### LINKED ISSUE #700: improve README recommended structure
"Short description of goals of package": underline how it should make it clear what it does, why one would care.

### LINKED ISSUE #701: de-duplicate info about badges in README section

### PR DESCRIPTION
**Pull request by maelle**
Fix #700 (added two question)
Fix #701 (there's an item further below on badge tables anyway)

### CODE REVIEW

File: pkg_building.es.Rmd [hunk 1]
```diff
@@ -148,8 +148,8 @@ bibentry(
...
-  - Una breve descripción de los objetivos del paquete con enlaces descriptivos a todas las viñetas (renderizadas, es decir, legibles, por ejemplo [el sitio web de documentación](#website)) a menos que el paquete sea pequeño y sólo haya una viñeta que repita el *README*.
+  - Etiquetas de integración continua y la cobertura de tests, la etiqueta de revisión por pares de rOpenSci una vez que haya comenzado (ver más abajo), una etiqueta de repostatus.org, y cualquier otra (por ejemplo [R-universe](https://ropensci.org/blog/2021/10/14/runiverse-badges/)).
+  - Una breve descripción de los objetivos del paquete (qué hace? ¿por qué es interesante?) con enlaces descriptivos a todas las viñetas (renderizadas, es decir, legibles, por ejemplo [el sitio web de documentación](#website)) a menos que el paquete sea pequeño y sólo haya una viñeta que repita el *README*.
```
Comment (yabellini) [inline pkg_building.es.Rmd:152]:
```suggestion
  - Una breve descripción de los objetivos del paquete (¿qué hace? ¿por qué es interesante?) con enlaces descriptivos a todas las viñetas (renderizadas, es decir, legibles, por ejemplo [el sitio web de documentación](#website)) a menos que el paquete sea pequeño y sólo haya una viñeta que repita el *README*.
```

**Review by yabellini — APPROVED** [review]
Small change.

**Review by yabellini — APPROVED** [review]

**Comment by maelle** [comment]
@mpadge you might have missed this one

**Comment by mpadge** [comment]
No, sorry, did see it, but I'd like to suggest a few edits. Just haven't had a chance yet. will do as soon as i can,

**Review by mpadge — APPROVED** [review]

**Comment by maelle** [comment]
@mpadge I edited the text, is it good so? Then I'll update the Spanish version. Thank you!

File: pkg_building.Rmd [hunk 2]
```diff
@@ -144,8 +144,8 @@ bibentry(
...
-  - Short description of goals of package, with descriptive links to all vignettes (rendered, i.e. readable, cf [the documentation website section](#website)) unless the package is small and there's only one vignette repeating the README.
+  - Badges for continuous integration and test coverage, the badge for rOpenSci peer-review once it has started (see below), a repostatus.org badge, and any other badges (e.g. [R-universe](https://ropensci.org/blog/2021/10/14/runiverse-badges/)).
+  - Short description of goals of package (what does it do? why should a potential user care?), with descriptive links to all vignettes unless the package is small and there's only one vignette repeating the README.  Please also ensure the vignettes are rendered, that is to say directly readable, cf [the documentation website section](#website)).
```
Comment (mpadge) [inline pkg_building.Rmd:148]:
```suggestion
  - Short description of goals of package (what does it do? why should a potential user care?), with descriptive links to all vignettes unless the package is small and there's only one vignette repeating the README.  Please also ensure the vignettes are rendered and readable, see [the "documentation website" section](#website)).
```

**Review by mpadge — APPROVED** [review]
Minor changes, but otherwise great

**Comment by maelle** [comment]
@maurolepore only if you have time and want to :pray:

File: pkg_building.es.Rmd [hunk 3]
```diff
@@ -148,8 +148,8 @@ bibentry(
...
-  - Una breve descripción de los objetivos del paquete con enlaces descriptivos a todas las viñetas (renderizadas, es decir, legibles, por ejemplo [el sitio web de documentación](#website)) a menos que el paquete sea pequeño y sólo haya una viñeta que repita el *README*.
+  - Etiquetas de integración continua y la cobertura de tests, la etiqueta de revisión por pares de rOpenSci una vez que haya comenzado (ver más abajo), una etiqueta de repostatus.org, y cualquier otra (por ejemplo [R-universe](https://ropensci.org/blog/2021/10/14/runiverse-badges/)).
+  - Una breve descripción de los objetivos del paquete (¿qué hace? ¿por qué seria interesante usarlo?) con enlaces descriptivos a todas las viñetas (renderizadas, es decir, legibles, por ejemplo [el sitio web de documentación](#website)) a menos que el paquete sea pequeño y sólo haya una viñeta que repita el *README*. Asegúrate también de que las viñetas estén renderizadas y sean legibles, consulta [la sección "sitio web de documentación"](#website))
```
Comment (maurolepore) [inline pkg_building.es.Rmd:152]: Looks like the final stop mark "." is missing.

**Review by maurolepore — APPROVED** [review]
LGTM. 
I only see one "." missing.

**Comment by maelle** [comment]
Thank you @maurolepore!!

**Pull request merged**